docs/static/transforming-data.asciidoc

```diff
@@ -455,25 +455,31 @@ The <<plugins-filters-http,http filter>> integrates with external web
 services/REST APIs, and enables lookup enrichment against any HTTP service or
 endpoint. This plugin is well suited for many enrichment use cases, such as
 social APIs, sentiment APIs, security feed APIs, and business service APIs.
-//+
-//[source,txt]
-//-----
-//filter {
-//  http {
-//    url => "http://example.com"
-//    verb => GET
-//    body => {
-//      "user-id" => "%{user}"
-//      "api-key" => "%{api_key}"
-//    }
-//    body_format => "json"
-//    headers =>
-//      "Content-type" => "application/json"
-//    }
-//    target_body => "new_field"
-//  }
-//}
-//-----
++
+[source,txt]
+-----
+http {
+    url => "https://neutrinoapi.com/convert"
```

Comment:
Would it make sense to generalize this example a bit? I also noticed the other examples have a short preamble describing the following code snippet, should we add this for http and memcached too?